.eslintrc.json

```diff
@@ -11,39 +15,17 @@
 	},
 	"plugins": ["@typescript-eslint"],
 	"rules": {
-		"brace-style": ["error", "stroustrup", { "allowSingleLine": true }],
-		"comma-dangle": ["error", "always-multiline"],
-		"comma-spacing": "error",
-		"comma-style": "error",
-		"curly": ["error", "multi-line", "consistent"],
-		"dot-location": ["error", "property"],
 		"handle-callback-err": "off",
-		"indent": ["error", "tab"],
 		"max-nested-callbacks": ["error", { "max": 4 }],
 		"max-statements-per-line": ["error", { "max": 2 }],
 		"no-console": "off",
 		"no-empty-function": "error",
-		"no-floating-decimal": "error",
 		"no-lonely-if": "error",
-		"no-multi-spaces": "error",
-		"no-multiple-empty-lines": ["error", { "max": 2, "maxEOF": 1, "maxBOF": 0 }],
 		"no-shadow": "off",
 		"@typescript-eslint/no-shadow": ["error"],
-		"no-trailing-spaces": ["error"],
 		"no-var": "error",
-		"object-curly-spacing": ["error", "always"],
 		"prefer-const": "error",
-		"quotes": ["error", "double"],
-		"semi": ["error", "always"],
```

Comment:
Lots of stuff removed in the eslint config, is this all added in the `extends prettier` line?

Reply:
Everything removed from the ESLint config is redundant; Prettier's default does the same thing.